lib/ledger_sync/adaptors/quickbooks_online/item/ledger_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Item
+        class LedgerSerializer < QuickBooksOnline::LedgerSerializer
```

Comment:
Is there not more to this object?

Reply:
We dont sync it. All we need is to reference relationship by ID.